group/definitions.js

```diff
@@ -0,0 +1,96 @@
+const messageId = {
+  type: 'string',
+  $comment: '42 chars + 1 char from a limited subset (last bits are 0) + 1 =',
+  pattern: '^ssb:message/classic/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$'
+}
+
+const groupId = {
+  type: 'string',
+  pattern: '^ssb:identity/group/[a-zA-Z0-9_\\-]{42}[AEIMQUYcgkosw048]=$'
```

Comment:
:hot_pepper:  this is gonna need to be more flexible for Ahau.... unless this is only needed for publishing new messages